src/test/remote/test_remote.py

```diff
@@ -0,0 +1,54 @@
+import yaml
+from subprocess import run, PIPE
+from unittest import TestCase
+from getpass import getuser
+from typing import Dict
+
+from ja.common.proxy.command_handler import CommandHandler
+
+
+class CommandHandlerDummy(CommandHandler):
+
+    def _process_command_string(
+            self, command_dict: Dict[str, object], type_name: str, username: str) -> Dict[str, object]:
+        assert username == getuser()
+        assert type_name == "COMMAND"
+        command_string = str(command_dict["payload"])
+        response_string = ""
+        while command_string.startswith("COMMAND"):
+            command_string = command_string[7:]
+            response_string += "RESPONSE"
+        return dict(payload=response_string)
+
+
+class RemoteTest(TestCase):
+    """
+    Class for testing Remote and CommandHandler.
+    IMPORTANT: THESE TESTS WILL FAIL IF YOU DON'T RUN THEM FROM THE BASE SRC DIRECTORY.
+    """
+
+    def _call_remote(self, command_string: str) -> str:
+        command_dict = dict(command=dict(payload=command_string), type_name=self._command_string)
+        finished_process = run(
+            [
+                "python3",
+                "test/remote/remote"  # Change to "remote" to run from current directory
```

Comment:
Could we have something like `python3 -m test.remote.remote` and require to just set `PYTHONPATH` ?